react/MiniCart.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { graphql } from 'react-apollo'
+import orderFormQuery from './graphql/orderFormQuery.gql'
+import MiniCartItem from './MiniCartItem'
+import Button from '@vtex/styleguide/lib/Button'
+import { Price } from '@vtex/product-details'
+import Spinner from '@vtex/styleguide/lib/Spinner'
+import './global.css'
+
+/**
+ * Minicart component
+ */
+class MiniCart extends Component {
+  handleClickButton = () => location.assign('/checkout/#/cart')
+
+  renderWithoutProducts = (label) => (
+    <div className="vtex-minicart__item pa4 shadow-4 flex items-center justify-center">
+      <span className="f5">{label}</span>
+    </div>
+  )
+
+  renderMiniCartWithItems = (orderForm, label) => (
+    <div className="flex flex-column">
+      <div className="vtex-minicart__arrow-up self-end mr3 pr1"></div>
+      <div className="shadow-3">
+        <div className="pa4 overflow-auto vtex-minicart__content">
```

Comment:
Should be the first class `vtex-minicart__content`

Reply:
Done!